gcp/README.md

```diff
@@ -138,6 +138,8 @@ Deploying to `stg` verifies that the gpii-infra code that worked to create a `de
 
 Because `stg` emulates production, it will (in the future) allow us to run realistic end-to-end tests before deploying to `prd`.
 
+The `stg` environment is almost the same environment as `prd`, this fact makes `stg` the proper environment to test some recovering procedures that the ops team must perform periodically. Like the restoration of the persistance layer. **The ops team will perform a restoration of the persistance layer every first monday of the month as part of the procedures verification needed for the FERPA compliance, so expect an outage of about 30 min at that time of the services in `stg`**.
```

Comment:
I would unbold -- bolding a long sentence adds visual clutter; it's a minor point, not worth drawing a reader's attention.

Reply:
I would move most of this under the last bullet point in the SLO section (this is what I requested yesterday).